examples/message-ubench/main.pony

```diff
@@ -185,13 +215,21 @@ actor SyncLeader
       let rate: I64 = (_partial_count.i64() * 1_000_000_000) / run_ns
       @printf[I32]("%lld,%lld\n".cstring(), run_ns, rate)
 
-      (let t_s: I64, let t_ns: I64) = Time.now()
-      tell_all_to_go(_ps, _initial_pings)
-      _total_count = _total_count + _partial_count
-      _last_t = to_ns(t_s, t_ns)
-      _waiting_for = _ps.size()
+      if not _run_done then
+        (let t_s: I64, let t_ns: I64) = Time.now()
+        tell_all_to_go(_ps, _initial_pings)
+        _total_count = _total_count + _partial_count
+        _last_t = to_ns(t_s, t_ns)
+        _waiting_for = _ps.size()
+      end
     end
 
+  be run_done(count: U64) =>
```

Comment:
given that `count` is never used, it can be removed.

Reply:
fixed